pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["flit_core >=3.9,<4"]
+requires = ["flit_core >=3.10,<4"]
```

Comment:
This isn't a Python version it's the flt version, we could reasonably update to `3.12` though I'd like it to be in a separate commit if we do.